src/pages/VenuePage/VenuePage.tsx

```diff
@@ -200,6 +201,20 @@ const VenuePage = () => {
     };
   }, [prevLocations, user, venueName]);
 
+  useEffect(() => {
+    if (
```

Comment:
nit: these returns can be combined

Reply:
It also looks like it could be simplified to `profile?.enteredVenueIds?.includes(venue?.id)`

Reply:
Also, nit, a newline of space after a 'guard' statement/pattern like we're using here makes things much clearer/easier to read.

Reply:
It looks like we also want to check `!profile` in the guard too. Would simplify the latter statement.